.npm/couper-binary.js

```diff
@@ -34,15 +34,16 @@ function getPlatform() {
 class CouperBinary {
 
 	constructor() {
-		// FIXME
-		let { version, name, repository } = require("./package.json")
+		let { version } = require("./package.json")
 		version = "1.7.1"
```

Comment:
seems to be hardcoded ?

Reply:
Yeah, until we get in sync with the package version. :unamused: